Nie-ontologies/Generic-ontologies/linguistic-morphology.ttl

```diff
@@ -201,28 +201,28 @@ lingmor:IndicativeMood
 lingmor:InjuncitveMood
 	rdfs:isDefinedBy <http://e-editiones.ch/ontology/linguistic-morphology#>;
 	a rdfs:Class;
-	rdfs:label "injunctive mood"@en, "Injunktiv"@de;
+	rdfs:label "injunctive mood"@en, "Injunktiv"@de;#tbd: injuctive = ? 
 	rdfs:comment """Verbal mood ... ."""@en;
 	rdfs:subClassOf lingmor:VerbalMood.
 
 lingmor:OptativeMood
 	rdfs:isDefinedBy <http://e-editiones.ch/ontology/linguistic-morphology#>;
 	a rdfs:Class;
-	rdfs:label "optative mood"@en, "Optativ"@de;
+	rdfs:label "optative mood"@en, "optimistisch"@de;
 	rdfs:comment """Verbal mood indicating wishes."""@en;
 	rdfs:subClassOf lingmor:VerbalMood.
 	
 lingmor:PrecativeMood
 	rdfs:isDefinedBy <http://e-editiones.ch/ontology/linguistic-morphology#>;
 	a rdfs:Class;
-	rdfs:label "precative mood"@en, "Präkativ"@de;
+	rdfs:label "precative mood"@en, "Präkativ"@de;#tbd: precative = ? 
```

Comment:
'Präkativ' is OK (Reto: rare in German as it is a phenomenon in Semitic languages).